src/sidenav/index.css

```diff
@@ -55,6 +55,10 @@
     border: var(--spectrum-tabs-focus-ring-size) solid transparent;
     border-radius: var(--spectrum-sidenav-item-border-radius);
   }
+
+  .spectrum-SideNav-itemIcon {
+    margin-right: 8px;
```

Comment:
I am not sure about how to create a DNA token for this. @GarthDB/@lazd could you point me in the right direction?

Reply:
@xitter basically you'd send a PR to spectrum-dna adding `icon-gap` to this file: https://git.corp.adobe.com/Spectrum/spectrum-dna/blob/master/data/elements/sidenav/SidenavBase.js#L46

 That said, it's already there, so you can simply reference `--spectrum-sidenav-icon-gap` here.